src/Apps/Order/Routes/Payment/index.tsx

```diff
@@ -184,22 +227,50 @@ export const PaymentRoute: FC<PaymentRouteProps> = props => {
     }
   }
 
-  // pushes to the review step, when payment is set with an existing bank account
-  const onBankAccountContinue = () => {
-    props.router.push(`/orders/${props.order.internalID}/review`)
-  }
-
   // fired when balance check is done: either sets error state or moves to /review
-  const onBalanceCheckComplete = (displayInsufficientFundsError: boolean) => {
+  const onBalanceCheckComplete = (
```

Comment:
I know this code was here before, but its important to call out for clarity sake. Idiomatically this form is also incorrect per long-term patterns in the react community: 

Local functions that _handle_ actions are always prefixed with `handle`, eg: `handleClick`:

```tsx
const Foo = () => {
  const handleClick = () => {
    ... 
  }
  
  return <Box onClick={handleClick} />
}
```

Or even

```tsx
const Foo = () => {
  const handleClick = () => {
    ... 
  }
  
  return <Bar onClick={handleClick} />
}

const Bar = ({ onClick }) => {
  const [isLoading, setIsLoading] = useState(false) 
  
  const handleClick = () => {
      setIsLoading(true)
      onClick?.() 
  }
  
  return <Box onClick={handleClick}
}
```

The rule is: 
- callback functions passed _down_ to components are defined as `onActionFoo` (think: `onClick`, `onMouseDown`, etc -- old patterns well defined in js community)
- local functions that _handle_ callback related actions are prefixed with `handleActionFoo`